src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -76,6 +76,19 @@ alibabacloud() {
 gcp() {
     test_vars GCP_CREDENTIALS GCP_PROJECT_ID GCP_ZONE PODVM_IMAGE_NAME
 
+    # TODO: The following manual conversions are kept for compatibility but should
```

Comment:
Could you split this commit into individual ones, please? With proper git log description, explaining why each change is needed?